### PR TITLE
[RFR] SUI beginnings

### PR DESCRIPTION
**Pull request by sshveta**
SSUI login test
{{pytest: cfme/tests/test_login.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d5325074c5c8400988a031c2de259d82a999e406:

`cfme/base/ssui.py`:
- :grey_exclamation: Line 23:5: **T001** *print statement found.*

`cfme/base/ui.py`:
- :grey_exclamation: Line 31:5: **T001** *print statement found.*

`cfme/tests/ssui/test_ssui_login.py`:
- :red_circle: Line 2:1: **F401** *'cfme.web_ui.flash' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e97ab6b4085c82cbe503cc91213ab54f2bc8f234:

`cfme/base/ssui.py`:
- :grey_exclamation: Line 23:5: **T001** *print statement found.*

`cfme/base/ui.py`:
- :grey_exclamation: Line 31:5: **T001** *print statement found.*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 926e1a37663b19ffd132a700ec587d4942d7df94:

`cfme/base/ssui.py`:
- :grey_exclamation: Line 23:5: **T001** *print statement found.*

`cfme/base/ui.py`:
- :grey_exclamation: Line 31:5: **T001** *print statement found.*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/base/ssui.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+class LoginPage(View):
+    flash = FlashMessages('div#flash_text_div')
+    username = Input(id='inputUsername')
+    password = Input(id='inputPassword')
+    login = Button('Log In')
+
+
+@Server.address.external_implementation_for(ViaSSUI)
+def address(self):
+    print "USING SSUI ADDRESS"
```
Comment (quarckster) [inline cfme/base/ssui.py]: **REQUIRED** please use `logger` here
```python
from utils.log import logger
```

File: cfme/base/ui.py [hunk 2]
```diff
@@ -8,16 +12,213 @@
...
 from utils.appliance.implementations.ui import navigator, CFMENavigateStep, ViaUI, navigate_to
 from . import Server, Region, Zone, ZoneCollection
 
+from utils import conf
+from utils.log import logger
+
+
+@Server.address.external_implementation_for(ViaUI)
+def address(self):
+    print "USING UI ADDRESS"
```
Comment (quarckster) [inline cfme/base/ui.py]: **REQUIRED** please use `logger` here

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by sshveta** [comment]
@psav :There is a test failure for bad_password test in 5.8 which looks like an issue because of difference in flash locator in 5.8 from 5.7 . Looks like there is already a PR to fix that issue.

**Pull request merged**